8/root/pre-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [[ $OUTRIGGER_QUIET == "0" ]]; then
+  echo "Running Node"
+  echo "Node: $(node --version); npm: $(npm --version); yarn: $(yarn --version)"
+  echo
+fi
```

Comment:
So what is the purpose of this? Just to output for environmental awareness?

Reply:
Yes, exactly.